### PR TITLE
Remove font size feature

### PR DESCRIPTION
**Pull request by shucon**
https://github.com/phpmyadmin/phpmyadmin/issues/13919
Signed-off-by: Saksham Gupta <shucon01@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

**Comment by mauriciofauth** [comment]
Apparently you just remove the font size form. The font size feature goes beyond that.

**Comment by shucon** [comment]
@mauriciofauth is this good?

**Comment by mauriciofauth** [comment]
In addition to removing the form, I think removing the [`FontSize` configuration directive](https://docs.phpmyadmin.net/en/latest/config.html#cfg_FontSize) is also necessary.

**Comment by devenbansod** [comment]
Hi @shucon I tried restarting the selenium job but the same test `PhpMyAdmin\Tests\Selenium\SettingsTest::testHideDatabase` failed again. Attached is the screen recording of them same. 

I'd recommend you to try running the test locally (or run down the flow manually), if it fails that'd mean that your changes in this PR unknowingly seem to have broken the flow being tested in the above test or the test has been broken since a last few commits but no one caught it.

bs-video-logs-use-2-

**Comment by devenbansod** [comment]
Also, I can see an error on the settings page saying: 'Field FontSize has no type' which seems related to this PR.

**Comment by shucon** [comment]
@devenbansod both the issues are fixed on my side , can you please confirm the same .

File: test/classes/ConfigTest.php [hunk 1]
```diff
@@ -80,71 +80,6 @@ public function testCheckSystem()
...
-            Config::getFontsizeForm()
-        );
-
-        $this->assertContains(
-            '<label for="select_fontsize">',
-            Config::getFontsizeForm()
-        );
-
-        //test getFontsizeOptions for "em" unit
-        $fontsize = $GLOBALS['PMA_Config']->get('FontSize');
```
Comment (mauriciofauth) [inline test/classes/ConfigTest.php]: There are more places where the _FontSize_ setting is used with the Config class.

**Comment by mauriciofauth** [comment]
This pull request is incomplete, there are still several places where the FontSize configuration is used.

**Comment by shucon** [comment]
@mauriciofauth is this sufficient?

File: themes/pmahomme/css/common.css.php [hunk 2]
```diff
@@ -14,9 +14,6 @@
 ?>
 /******************************************************************************/
 /* general tags */
-html {
-    font-size: <?php echo $theme->getFontSize(); ?>
-}
```
Comment (mauriciofauth) [inline themes/pmahomme/css/common.css.php]: You need to keep the font-size property here.

File: libraries/classes/Config.php [hunk 3]
```diff
@@ -1191,13 +1191,7 @@ public function getSource()
      */
     public function getThemeUniqueValue()
```
Comment (mauriciofauth) [inline libraries/classes/Config.php]: You should update the doc for this method to reflect the changes.

**Comment by mauriciofauth** [comment]
I don't think it's a good idea to include this in 4.8 branch. This is a major change and should only be included in 5.0 branch.

File: libraries/classes/Config.php (hunk 3 above)
Comment (shucon) [reply libraries/classes/Config.php]: I can't find anything related to this in the doc, please help.

**Comment by shucon** [comment]
@mauriciofauth everything good now?

**Review by MauricioFauth — APPROVED** [review]

File: doc/config.rst [hunk 4]
```diff
@@ -3228,13 +3228,6 @@ Theme manager settings
 
     Whether to allow different theme for each server.
 
-.. config:option:: $cfg['FontSize']
-
-    :type: string
-    :default: '82%'
-
-    Font size to use, is applied in CSS.
-
```
[the remark this answers is not in the document]
Comment (shucon) [reply doc/config.rst]: Okay, got it! I'll fix it.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Pull request merged**